src/java/org/apache/cassandra/cql3/statements/schema/CreateTableStatement.java

```diff
@@ -399,6 +399,16 @@ else if (!builder.hasRegularColumns())
             warnings.add("The unsupported 'nodesync' table option was ignored.");
         }
 
+        if (attrs.hasProperty("dse_vertex_label_property"))
```

Comment:
Given we have IGNORED_KEYWORDS should we just make this code generic looping over IGNORED_KEYWORDS and adding the warning?  So we just need to add something new to IGNORED_KEYWORDS in the future and not need to also add new warnings here etc?

Reply:
I haven't gone with this approach because not all of the properties come from the user. Some of them are added to attrs to signal some specific structures in the statement. As a result, the user may be confused by the generic WARN.

E.g. specifying VERTEX LABEL keywords in CREATE TABLE stmt results in "dse_vertex_label_property" added to attrs.  A warning like "The unsupported graph table property was ignored (dse_vertex_label_property)" might not be clear for some.
Of course we might keep a mapping property -> human readable name but I thought it's an overkill.

Let me know what you think.

Reply:
Got it.  Yeah that makes sense.